cmd/juju/commands/bootstrap.go

```diff
@@ -329,7 +329,6 @@ func (c *bootstrapCommand) Run(ctx *cmd.Context) (resultErr error) {
 	if err := c.parseConstraints(ctx); err != nil {
 		return err
 	}
-
```

Comment:
please revert style changes (here and below in this file), or send a separate PR with justification